applications/software-libraries.md

```diff
@@ -0,0 +1,59 @@
+# Software Libraries
+
+The following describes the approach we adopt when making use of external software libraries in our applications.
+
+## Centrally capture your application's dependencies *in code*
+
```

Comment:
Can we clarify: does this apply only to libraries directly "linked" to the main "executable", or all dependencies (which would include services like Address Lookup, and often a Database)?  Gemfile / POM / package.js would only capture only the first subset.